test/cpp/util/spiffe_test_credentials.h

```diff
@@ -0,0 +1,56 @@
+/*
+ *
+ * Copyright 2019 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef GRPC_TEST_CPP_UTIL_SPIFFE_TEST_CREDENTIALS_H
+#define GRPC_TEST_CPP_UTIL_SPIFFE_TEST_CREDENTIALS_H
+
+#include <grpcpp/security/credentials.h>
+#include <grpcpp/security/server_credentials.h>
+#include <memory>
+#include <thread>
+#include <vector>
+
+namespace grpc {
+namespace testing {
+
+const char kSpiffeCredentialsType[] = "tls";
```

Comment:
Please also rename the file to `tls_test_credentials.h`.

Reply:
Done. But note that I can't rename `kSpiffeCredentialsType` to `kTlsCredentialsType`, because you already renamed the latter from `kSslCredentialsType`.